client/my-sites/plan-features/style.scss

```diff
@@ -10,6 +10,12 @@ $plan-features-sidebar-width: 272px;
 	// If experiment succeeds the following styles
 	// need to be moved to the corresponding selectors
 
+	.plans-features-main__group.is-wpcom {
+		overflow: auto;
+		max-height: calc( 100vh - 115px );
```

Comment:
Nit; might wanna add note where magic number `115px` comes from, or have it as a variable if it's used elsewhere, too.